enacts/config-sng.yaml

```diff
@@ -52,21 +57,56 @@ onset:
         monit:
             menu_label: Has germinating rain occured?
             description: The map shows the result of the germinating rain date search
-            (i.e. near-real time thus without checking of follow-up dry spells)
-            from the most recent Early Start to now
-            (or the last day with available rainfall data).
-            Dates are expressed in days since an Early Start date.
+                (i.e. near-real time thus without checking of follow-up dry spells)
+                from the most recent Early Start to now
+                (or the last day with available rainfall data).
+                Dates are expressed in days since an Early Start date.
         mean:
             menu_label: When to prepare for planting?
             description: The map shows the average onset date over all years of available data.
-            Dates are expressed in days since an Early Start date.
+                Dates are expressed in days since an Early Start date.
         stddev:
             menu_label: How uncertain is the planting date?
             description: The map shows the standard deviation from the average onset date.
-            over all years of available data.
-            Dates are expressed in days since an Early Start date.
+                over all years of available data.
+                Dates are expressed in days since an Early Start date.
         pe:
             menu_label: How risky to plant...
             description: The map shows the probability of the onset date to be past a certain date,
-            through all the years of available data.
-            Dates are expressed in days since an Early Start date.
+                through all the years of available data.
+                Dates are expressed in days since an Early Start date.
+
+# Water Balance Monitoring
+wat_bal_monit:
+
+    # Data
+    rr_mrg_nc_path: /Data/data23/ANACIM/anacim_v3/ALL_block2/rr_mrg_daily/
```

Comment:
Different apps use different data: onset uses daily precip (ALL and MON -- uses actually only ALL for now until we get some daily MON available); flex fcst doesn't use ENACTS data; monthly uses monthly precip/tmin/tmax ALL; wat_bal_monit uses daily precip/tmin/tmax MON is possible, if not ALL; wat_bal_history will use daily precip/tmin/tmax ALL.

Plus I was understanding that the chunking when converting to zar could also be app specific...

So can we really have that be in top level config?

Reply:
I say let's put it in the top level until we have a need to differentiate. We might eventually have to use different zarrifications for different pages, but if we can avoid it that would be preferable, so let's start with that assumption. As for different pages using different variables, there aren't that many variables in ENACTS so I assume that pretty soon we'll be using all of them, some on more than one page.

Reply:
done but I am not resolving because I feel there will be more to do about this.

Reply:
I also made one enactstozarr file. It's dirty...

Reply:
Indeed, it's dirty. It would be nice to have one config entry that provides all the enacts dataset info, with a consistent structure.

I'm making the scope of this PR creep larger and larger, just because I wasn't reviewing as carefully when we were in a rush ahead of the training. We can make an issue for this and fix it later.

Reply:
issue created